qutebrowser/javascript/scroll.js

```diff
@@ -22,53 +22,98 @@
 window._qutebrowser.scroll = (function() {
     const funcs = {};
 
-    funcs.to_perc = (x, y) => {
-        let x_px = window.scrollX;
-        let y_px = window.scrollY;
+    const call_if_frame = window._qutebrowser.utils.call_if_frame;
```

Comment:
I have the same issue I have in Python when people import stuff from modules - I always think `call_if_frame` must be in this file somewhere :wink:. What about having a `const utils = window._qutebrowser.utils;` and then using `utils.call_if_frame` instead?